HW 5 (mock-data + JS)/Task 4/JS/index.js

```diff
@@ -0,0 +1,15 @@
+function getWord() { 
+    const arr1 = ['e', null, 'r'];
+    const arr2 = ['g', 'y'];
+
+    const newArray = arr1.concat(arr2);
+    newArray.unshift('p', 'o');
+    newArray.push('m', 'o', 'd', 'e');
+    newArray.splice(2, 0, 'w');
+    newArray.splice(6, 2);
+    newArray.splice(6, 1, 'c');
+    let newArrayFilter = newArray.filter((value) => value);
+    let newArrayJoin = newArrayFilter.join(' ');
```

Comment:
`використовуючи функцію arrayJoin об'єднати елементи масиву в одне слово`
Використайте в якості розділювача пустий рядок.
```suggestion
    return arrayJoin(newArrayFilter, '');
```